pkg/cmd/cli/cmd/newapp.go

```diff
@@ -191,31 +192,27 @@ func RunNewApplication(fullName string, f *clientcmd.Factory, out io.Writer, c *
 		return err
 	}
 
-	if len(cmdutil.GetFlagString(c, "output")) != 0 {
+	if len(output) != 0 && output != "name" {
 		return f.Factory.PrintObject(c, result.List, out)
 	}
-	if err := createObjects(f, out, result); err != nil {
+	if err := createObjects(f, out, output == "name", result); err != nil {
 		return err
 	}
 
 	hasMissingRepo := false
 	for _, item := range result.List.Items {
 		switch t := item.(type) {
-		case *kapi.Service:
-			portMappings := "."
-			if len(t.Spec.Ports) > 0 {
-				portMappings = fmt.Sprintf(" with port mappings %s.", describeServicePorts(t.Spec))
-			}
-			fmt.Fprintf(c.Out(), "Service %q created at %s%s\n", t.Name, t.Spec.ClusterIP, portMappings)
 		case *buildapi.BuildConfig:
-			fmt.Fprintf(c.Out(), "Build configuration %q created and build triggered.\n", t.Name)
+			if len(t.Spec.Triggers) > 0 {
+				fmt.Fprintf(c.Out(), "Build scheduled for %q - use the build-logs command to track its progress.\n", t.Name)
+			}
 		case *imageapi.ImageStream:
 			if len(t.Status.DockerImageRepository) == 0 {
 				if hasMissingRepo {
 					continue
 				}
 				hasMissingRepo = true
-				fmt.Fprintf(c.Out(), "WARNING: We created an image stream %q, but it does not look like a Docker registry has been integrated with the server. Automatic builds and deployments depend on that integration to detect new images and will not function properly.\n", t.Name)
+				fmt.Fprintf(c.Out(), "WARNING: No Docker registry has been configured with the server. Automatic builds and deployments may not function.\n", t.Name)
```

Comment:
Yeah, we probably just want to warn - they'll know how to ask their admin.

On Thu, Oct 15, 2015 at 8:46 AM, Cesar Wong <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/newapp.go
> <https://github.com/openshift/origin/pull/5136#discussion_r42119029>:
>
> >  		case *imageapi.ImageStream:
> >  			if len(t.Status.DockerImageRepository) == 0 {
> >  				if hasMissingRepo {
> >  					continue
> >  				}
> >  				hasMissingRepo = true
> > -				fmt.Fprintf(c.Out(), "WARNING: We created an image stream %q, but it does not look like a Docker registry has been integrated with the server. Automatic builds and deployments depend on that integration to detect new images and will not function properly.\n", t.Name)
> > +				fmt.Fprintf(c.Out(), "WARNING: No Docker registry has been configured with the server. Automatic builds and deployments may not function.\n", t.Name)
>
> The user of this command may not be the administrator. Might cause more
> confusion than good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5136/files#r42119029>.
>